sma1/shmem_both_put_nb_size.c

```diff
@@ -158,10 +162,17 @@ int main(int argc, char **argv)
        }
      shmem_barrier_all();
      if ( (my_pe % 2) == 0 ) {
+#ifndef OPENSHMEM
        shmem_int_put_nb(targ_int,srce_int,elements[i],my_pe+1,NULL);
        shmem_long_put_nb(targ_long,srce_long,elements[i],my_pe+1,NULL);
        shmem_float_put_nb(targ_float,srce_float,elements[i],my_pe+1,NULL);
        shmem_double_put_nb(targ_double,srce_double,elements[i],my_pe+1,NULL);
+#else
+       shmem_int_put_nbi(targ_int,srce_int,elements[i],my_pe+1);
```

Comment:
We can just test for the _nbi routines. All _nb routines are legacy code and they should also be shmemx_ prefixed.